CMU-313/nodebb-f24-team-kale
# Add UI for displaying question status as 'Resolved' or 'Not Resolved'

**Description:**
I implemented a UI feature in the General Discussion section that displays the resolved status of questions, based on backend logic. If a question is answered, it is marked as 'Resolved'. If not, it is marked as 'Not Resolved'.
The resolved status is displayed next to topics in the topic list.

**I made the following changes:**
1. {{{if}}}, {{{else}}}, {{{end}}}. for display backend logic
2. Added logic to display a status badge next to each topic in the topic list.
3. If the question is resolved, display a green "Resolved" badge. Else, display a gray "Not Resolved" badge.

**Overview:**
The resolved status badge is visible in the topic list, making it easier for users to identify which discussions have been resolved at a glance.
![Screenshot 2024-09-22 at 2 13 37 PM](https://github.com/user-attachments/assets/3fd7e772-bd11-46da-a963-02781252242f)


**Related Issues:**
#1 #2

**How to Test:**
1. Navigate to the General Discussion section.
2. For topics that have been answered, you should see a green "Resolved" badge.
3. For topics without answers, a gray "Not Resolved" badge should be displayed.

**Resolves #5**
